### PR TITLE
feat: Allow input number arrows overrides

### PR DESCRIPTION
**Pull request by aleiton**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [X] New feature
- [ ] Bug fix
- [X] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [X] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
Our current antd use case requires the default upArrow and downArrow to be modified. That is currently not possible.

### 💡 Background and solution

Solution: Hoist the default icons and allow for overrides.

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    `Input-Icon` now supports `customUpArrow` and `customDownArrow`   |
| 🇨🇳 Chinese |   `Input-Icon` 现在支持 `customUpArrow` 和 `customDownArrow`        |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [X ] Doc is updated/provided or not needed
- [ X] Demo is updated/provided or not needed
- [X] TypeScript definition is updated/provided or not needed
- [ X] Changelog is provided or not needed

### CODE REVIEW

File: components/input-number/index.en-US.md [hunk 1]
```diff
@@ -20,6 +20,8 @@ When a numeric value needs to be provided.
 | autoFocus | If get focus when component mounted | boolean | false | - |
 | bordered | Whether has border style | boolean | true | 4.12.0 |
 | controls | Whether to show `+-` controls | boolean | true | 4.17.0 |
+| customUpIcon | The custom icon displayed instead of the default up arrow icon | ReactNode | - |  |
```
Comment (xrkffgg) [inline components/input-number/index.en-US.md:23]: Add a demo?

Comment (aleiton) [reply components/input-number/index.en-US.md:23]: Got it, working on that.

Comment (aleiton) [reply components/input-number/index.en-US.md:23]: @xrkffgg I just added a demo, I translate the words to Chinese simplified with google translate, feel free to update that if it's wrong.

**Review by hamzawaleed0102 — APPROVED** [review]
Looks good to me!

**Comment by xrkffgg** [comment]
Please update snap

**Comment by R0nd** [comment]
I am interested in this feature and would be happy to see it merged 👍

**Comment by aleiton** [comment]
@zombieJ hi, I want to continue with this PR, as only update the snap is needed, how could we re-open it? Thanks

**Pull request closed without merging**